scrapper.py

```diff
@@ -58,6 +59,20 @@ def validate_config(crawler_path):
     pass
 
 
+def main():
+    response = requests.get('https://russkayarech.ru/ru')
```

Comment:
I see that you are able to download a page. But this is far not enough to hand over programming assignment